documentation/assemblies/configuring/assembly-storage.adoc

```diff
@@ -13,18 +13,24 @@ The supported storage types are:
 * Ephemeral (Recommended for development only)
 * Persistent
 * JBOD (Kafka only; not available for ZooKeeper)
+* Tiered storage (Early access preview)
 
 To configure storage, you specify `storage` properties in the custom resource of the component. 
 The storage type is set using the `storage.type` property.
-
-When using node pools, you can specify storage configuration unique to each node pool used in the cluster.
+When using node pools, you can specify storage configuration unique to each node pool used in a Kafka cluster.
 The same storage properties available to the `Kafka` resource are also available to the `KafkaNodePool` pool resource.
 
+Tiered storage provides more flexibility for data management by leveraging the use of both local storage and remote storage solutions.
```

Comment:
I don't think this is completely wrong. But it can be confusing by using the overloaded terms such as local and remote storage. Maybe this would be better?

```suggestion
Tiered storage provides more flexibility for data management by leveraging the use of storage types with different characteristics in parallel.
For example:
* Regular block storage with high performance and high cost
* Object storage with lower cost but also lower performance.
```